mpmath/tests/test_cli.py

```diff
@@ -26,7 +26,7 @@ def __del__(self):
 
 
 def test_bare_console_no_bare_division():
-    c = Console('python -m mpmath --no-ipython --no-wrap-floats')
+    c = Console('python3 -m mpmath --no-ipython --no-wrap-floats')
```

Comment:
It isn't always possible to depend on either of these. Instead `sys.executable` should be used.

Reply:
Sounds goods!  I'll push a new version of this PR with that suggestion.